content/docs/dispensary/prescriptions.md

```diff
@@ -464,12 +466,16 @@ Once the prescription has been confirmed:
 
 ## Printing Labels
 
-Open mSupply has basic label printing that will be built upon with more flexibility in the future.
+Open mSupply has basic label printing that will improved with more flexibility in the future.
 
 ![Prescription label](/docs/dispensary/images/prescription_label.jpg)
 ![Prescription abbreviation](/docs/dispensary/images/prescription_abbreviation.png)
 
-First ensure that you have configured a printer in [Settings](/docs/settings/devices/). Note that at the moment we use the ZPL printing language under the hood, which covers about half the hardware on the market. You'll need to tweak the printer configuration (directly on the device) to get it just right for our label printing.
+### Getting started
+
+To begin printing labels, first ensure that you have configured a printer in [Settings](/docs/settings/devices/).
+
+<div class="note">A technical note: at the moment we use the ZPL printing language under the hood, which covers about half the label printer hardware on the market. You'll need to tweak the printer configuration (directly on the device) to get it just right for our label printing. The support for more printers and label formats is coming in future releases of Open mSupply.</div>
```

Comment:
I thought that this was getting too technical for the average user. Didn't really want to highlight in a 'note' section but wanted to differentiate